runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/action.json

```diff
@@ -18,6 +18,7 @@
     "Action.Details.Dialog.LiveView": "PODGLĄD NA ŻYWO",
 
     
+    "Process.BuildView.Modeler.Widgets.VariableNameIsTaken": "Nazwa zmiennej już istnieje",
```

Comment:
imo it will be better this way `Zmienna o wybranej nazwie już istnieje`. Its important to inform about the variable of a given name not just a name